docker/.env

```diff
@@ -22,5 +22,16 @@ DWH_ROOT=./dwh
 # data dump set `small`, which is the default mode.
 DATABASE_DUMP_MODE=small
 
-# JVM parameters to be used by the application
-JAVA_OPTS=-Xms2g -Xmx2g
+# JVM parameters to be used by the application. If the application is set to use FlinkRunner for the
+# Beam pipelines in local cluster mode (which is the default currently), then the JVM total memory
+# should be roughly set with a minimum value derived by the below equation.
+
+# Minimum JVM Memory
+#   = Memory for JVM stack, perm files etc
+#     + (#Parallel Pipeline Threads * #Parallel Pipelines * (1.6 * Parquet Row Group Size))
```

Comment:
Is the constant `1.6` accurate or is it based on some estimate of how much memory is needed when the row-group is brought into memory? If it is the latter maybe use a slightly higher number and document it is an estimate.

Reply:
Yes, the `1.6` is an estimate, I have documented this and changed the value to be `2`